src/__tests__/queryByApi.test.js

```diff
@@ -72,6 +72,7 @@ test('queryByText with nested Text components return the closest Text', () => {
   const { queryByText } = render(<NestedTexts />);
 
   expect(queryByText('My text')?.props.nativeID).toBe('2');
+  expect(queryByText('My text', { exact: false })?.props.nativeID).toBe('2');
```

Comment:
but it's supported with `exact: false`, I think that's fine

Reply:
I'm not sure what you're talking about here. exact doesn't have anything to do with how we match text spread across multiple node (if this comment is linked to https://github.com/callstack/react-native-testing-library/pull/554/files/4087aff6f3cc2200762308911929a3de295902b4#r500085148).

Reply:
Ugh, I read it the wrong way 🙈